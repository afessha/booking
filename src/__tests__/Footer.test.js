import React from "react";
import { render } from "@testing-library/react";
import { create } from "react-test-renderer";

const Footer = () => {
  return <div className="footer"> Awet Fessha @2020 </div>;
};
export default Footer;

describe("Footer component", () => {
  test("Matches the snapshot", () => {
    const footer = create(<Footer />);
    expect(footer.toJSON()).toMatchSnapshot();
  });
});
