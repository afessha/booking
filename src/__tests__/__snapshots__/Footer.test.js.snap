// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Matches the snapshot 1`] = `
<div
  className="footer"
>
   Awet Fessha Â©2020 
</div>
`;
